uportal-war/src/main/resources/org/jasig/portal/portlets/SharedParameters.cpd.xml

```diff
@@ -43,38 +43,19 @@
         </parameter>
 
         <parameter>
-            <name>showChrome</name>
-            <label>show.portlet.window.chrome</label>
-            <single-choice-parameter-input display="select">
-                <default>true</default>
-                <option value="false" label="false"/>
-                <option value="true" label="true"/>
-            </single-choice-parameter-input>
-        </parameter>
-
-        <parameter>
             <name>alternativeMaximixedLink</name>
             <label>open.alternative.url.in.maximized.view</label>
             <single-text-parameter-input display="text"/>
         </parameter>
 
         <parameter>
-            <name>alternate</name>
-            <label>use.alternate.appearance</label>
-            <single-choice-parameter-input display="select">
-                <default>false</default>
-                <option value="false" label="false"/>
-                <option value="true" label="true"/>
-            </single-choice-parameter-input>
-        </parameter>
```

Comment:
I don't see a deletion of alternate from content.xsl.  Does this mean we are still supporting it (for backwards compatibility) but not displaying it?  What happens if you edit an existing portlet definition that had alternate set?